src/style.css

```diff
@@ -1 +1,67 @@
-/* Styles go here */
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
do not forget to add more fallback fonts
![image](https://github.com/mate-academy/layout_moyo-header/assets/22824431/d36031d8-4dde-404b-bf8a-e28525a940db)